Sources/MuxUploadSDK/Internal Utilities/ProcessInfo+EnvironmentVariables.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  File.swift
+//  
+//
+//  Created by Liam Lindner on 3/15/23.
+//
+
+import Foundation
+
+extension ProcessInfo {
+
+    /// Set the Sentry DSN an environment variable to track upload statistics
+    var sentryDsn: String {
+        environment[
+            "SENTRY_DSN"
```

Comment:
How is it intended that we add the DSN to the env? If users have to add it to their app plist from our docs, we might be better off just baking it in